druzhba/table.py

```diff
@@ -54,14 +54,12 @@ class MigrationError(Exception):
 
 
 class Permissions(namedtuple("Permissions", ["name", "is_group", "grants", "owner"])):
-    all_str = "arwdRxt"
-    all_grants = "ALL PRIVILEGES"
     char_to_grant = {
         "r": "SELECT",  # read
         "w": "UPDATE",  # write
         "a": "INSERT",  # append
         "d": "DELETE",
-        "D": "TRUNCATE",
+        "D": "DROP",
```

Comment:
`D` means `truncate` in postgres but was seemingly not implemented in Redshift until recently